  0%|          | 0/2000 [00:00<?, ?it/s]  0%|          | 0/2000 [00:02<?, ?it/s]
OSError: [Errno 122] Disk quota exceeded

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/exports/eddie/scratch/s2517783/mamba/tsp.py", line 135, in <module>
    tours_train, sumLogProbOfActions = seq2seq_generate_tour(device,model_train,inputs,deterministic=False)
                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/mamba/model.py", line 260, in seq2seq_generate_tour
    outputs = model(inputs)[:,-1,:]
              ^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/miniconda3/envs/tspp/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/miniconda3/envs/tspp/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/mamba/model.py", line 230, in forward
    x , residual = layer(x,residual)
                   ^^^^^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/miniconda3/envs/tspp/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/miniconda3/envs/tspp/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/mamba/mamba_ssm/modules/block.py", line 57, in forward
    hidden_states, residual = layer_norm_fn(
                              ^^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/mamba/mamba_ssm/ops/triton/layer_norm.py", line 902, in layer_norm_fn
    return LayerNormFn.apply(
           ^^^^^^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/miniconda3/envs/tspp/lib/python3.12/site-packages/torch/autograd/function.py", line 598, in apply
    return super().apply(*args, **kwargs)  # type: ignore[misc]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/mamba/mamba_ssm/ops/triton/layer_norm.py", line 775, in forward
    y, y1, mean, rstd, residual_out, seeds, dropout_mask, dropout_mask1 = _layer_norm_fwd(
                                                                          ^^^^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/mamba/mamba_ssm/ops/triton/layer_norm.py", line 369, in _layer_norm_fwd
    _layer_norm_fwd_1pass_kernel[(M,)](
  File "/exports/eddie/scratch/s2517783/miniconda3/envs/tspp/lib/python3.12/site-packages/triton/runtime/jit.py", line 167, in <lambda>
    return lambda *args, **kwargs: self.run(grid=grid, warmup=False, *args, **kwargs)
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/miniconda3/envs/tspp/lib/python3.12/site-packages/triton/runtime/autotuner.py", line 143, in run
    timings = {config: self._bench(*args, config=config, **kwargs) for config in pruned_configs}
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/miniconda3/envs/tspp/lib/python3.12/site-packages/triton/runtime/autotuner.py", line 122, in _bench
    return do_bench(kernel_call, warmup=self.warmup, rep=self.rep, quantiles=(0.5, 0.2, 0.8))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/miniconda3/envs/tspp/lib/python3.12/site-packages/triton/testing.py", line 102, in do_bench
    fn()
  File "/exports/eddie/scratch/s2517783/miniconda3/envs/tspp/lib/python3.12/site-packages/triton/runtime/autotuner.py", line 110, in kernel_call
    self.fn.run(
  File "/exports/eddie/scratch/s2517783/miniconda3/envs/tspp/lib/python3.12/site-packages/triton/runtime/autotuner.py", line 305, in run
    return self.fn.run(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/miniconda3/envs/tspp/lib/python3.12/site-packages/triton/runtime/autotuner.py", line 305, in run
    return self.fn.run(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/miniconda3/envs/tspp/lib/python3.12/site-packages/triton/runtime/autotuner.py", line 305, in run
    return self.fn.run(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/miniconda3/envs/tspp/lib/python3.12/site-packages/triton/runtime/jit.py", line 416, in run
    self.cache[device][key] = compile(
                              ^^^^^^^^
  File "/exports/eddie/scratch/s2517783/miniconda3/envs/tspp/lib/python3.12/site-packages/triton/compiler/compiler.py", line 197, in compile
    metadata_group[metadata_filename] = fn_cache_manager.put(json.dumps(metadata, default=vars), metadata_filename,
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/exports/eddie/scratch/s2517783/miniconda3/envs/tspp/lib/python3.12/site-packages/triton/runtime/cache.py", line 123, in put
    with open(temp_path, mode) as f:
OSError: [Errno 122] Disk quota exceeded
